packages/client/src/api/wargames_api/index.ts

```diff
@@ -152,42 +150,50 @@ export const listenForWargameChanges = (name: string, dispatch: PlayerUiDispatch
   listenNewMessage({ db, name, dispatch })
 }
 
-export const pingServer = (activityDetails: { wargame: string, role: string }): Promise<any> => {
+export const pingServer = async (wargame: string, role: string, activityTypes: string, activityTimes: string): Promise<any> => {
   const activityMissing = 'The player has not shown any activity yet'
-  const activityTime = encodeURIComponent(expiredStorage.getItem(`${activityDetails.role}_${ACTIVITY_TIME}`) || activityMissing)
-  const activityType = encodeURIComponent(expiredStorage.getItem(`${activityDetails.role}_${ACTIVITY_TYPE}`) || activityMissing)
-
-  const activityUrl = `${activityDetails.wargame || 'missing'}/${activityDetails.role || 'missing'}/${activityTime}/${activityType}`
-
-  return fetch(`${serverPath}healthcheck/${activityUrl}/healthcheck`, {
-    method: 'GET',
-    headers: {
-      'Content-Type': 'application/json'
-    }
-  })
-    .then((response: Response): Promise<any> => response.json())
-    .then((data: any) => {
-      return data.status
-    })
-    .catch((err) => {
-      console.log(err)
-      return 'NOT_OK'
-    })
-}
-
-export const getPlayerActivityLogs = () => {
-  const url = `${serverPath}playerlog`
-
-  return fetch(url, {
-    method: 'GET',
-    headers: {
-      'Content-Type': 'application/json'
-    }
-  })
-    .then((response: Response): Promise<any> => response.json())
-    .then((data: any) => {
-      return data
-    })
+  const activityTime = activityTimes || activityMissing
+  const activityType = activityTypes || activityMissing
+  
+  if (!wargame) return null
+  
+  const { db } = getWargameDbByName(wargame)
```

Comment:
@lilitkarapetyan  - I think you had a mistake in the above line.  I think you were trying to get `player-logs`, but that's not a wargame database